frontend/src/newConsole/components/newLogos/index.ts

```diff
@@ -1,2 +1 @@
-export * from './Logos';
```

Comment:
@spokerman12 since we are keeping the `Logos` component, please do export here.

Reply:
Thanks for noticing this.